src/runtime/runtime_atmega.go

```diff
@@ -1,4 +1,4 @@
-// +build avr,atmega
+// +build avr,atmega avr,atmega2560
```

Comment:
This change is not necessary with the addition of the `atmega` build tag.

Reply:
> > It might make sense to change our naming scheme a bit. I propose that we change `arduino` -> `arduino-uno`,
> 
> Yes, I think changing the name of the current `arduino` target is a good idea (I've often thought about it but never done it). We could leave an alias for `arduino` but use `arduino-uno` in all examples.
> 
> > and call this target `arduino-mega`.
> 
> Actually, there was an "Arduino Mega" based on the atmega1280. They now call it the "Arduino Mega 2560" to distinguish the two boards. Therefore I think it should be named `arduino-mega2560`.



> > It might make sense to change our naming scheme a bit. I propose that we change `arduino` -> `arduino-uno`,
> 
> Yes, I think changing the name of the current `arduino` target is a good idea (I've often thought about it but never done it). We could leave an alias for `arduino` but use `arduino-uno` in all examples.
> 
> > and call this target `arduino-mega`.
> 
> Actually, there was an "Arduino Mega" based on the atmega1280. They now call it the "Arduino Mega 2560" to distinguish the two boards. Therefore I think it should be named `arduino-mega2560`.

Should this type of renaming be done in this same PR?
I'll make them separate commits at the very least.

Reply:
> > Actually, there was an "Arduino Mega" based on the atmega1280. They now call it the "Arduino Mega 2560" to distinguish the two boards. Therefore I think it should be named `arduino-mega2560`.
> 
> Should this type of renaming be done in this same PR?
> I'll make them separate commits at the very least.

The Arduino Mega, yes. But not `arduino` -> `arduino-uno`, that's a separate matter.